modules/boss/src/main/ts/ephox/boss/api/Main.ts

```diff
@@ -6,7 +6,6 @@ import { SpecialGene } from './SpecialGene';
 import { TestUniverse } from './TestUniverse';
 import { TextGene } from './TextGene';
 import { Universe } from './Universe';
-
 /* eslint-disable import/order */
```

Comment:
Unfortunately no as it turns out. The disable only applies to the lines within the "ignore" block. So since `import { Universe } from './Universe';` is outside that, it'll still get flagged as it sees an import on the lines below and it doesn't match the spacing rules.

I mentioned a few alternatives on slack, but all are gross so I believe this is still the best solution.

Reply:
Yeah, I'm happy with this given the alternatives.